Index.html

```diff
@@ -32,8 +32,53 @@ <h2>I’m a software developer</h2>
          <li><a><img src="media/github.svg" alt="github link"></a></li>
          <li><a><img src="media/angellist.svg" alt="angellint link"></a></li>
         </ul>
+
+        <a class="disable-button"><img src="media/Disabled.svg" alt=""></a>
  
       </div>
+      <section class="work-container">
+        <div class="my-work-text">
+          <h2>My Recent Works</h2>
+          <hr>
+       </div>
+        <div class="work-history">
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
```

Comment:
- [ ] The goal of this section is to help students practice CSS Layout using **CSS Grid and Flexbox**. So instead of using screenshots of Figma's design, please implement them using HTML and CSS.